pallets/identity/src/keys.rs

```diff
@@ -244,6 +244,84 @@ impl<T: Config> Module<T> {
         Ok(())
     }
 
+    /// Accepts a primary key rotation.
+    /// Differs from accept_primary_key_rotation in that it directly swaps a primary key with a secondary key,
+    /// instead of replacing the primary key with an unlinked key, so it requires no authorization.
```

Comment:
"so it requires no authorization" is this comment correct? I believe this code first requires the user primary key to create and authorization of type `DirectRotatePrimaryKey` before the it is accepted with `rotate_primary_key`. So the primary key has given authorization and new primary confirmed.

Reply:
Oops forgot to update this comment after a previous iteration